test/e2e/framework/helper/featureset/featureset.go

```diff
@@ -126,4 +126,8 @@ const (
 	// OnlySAN denotes whether the target issuer is able to sign certificates
 	// with only SANs set
 	OnlySAN = "OnlySAN"
+
+	// SaveCAToSecretFeature denotes whether the target issuer returns a CA
+	// certificate which can be stored in the CA.crt field of the Secret.
+	SaveCAToSecretFeature Feature = "SaveCAToSecretFeature"
```

Comment:
Previously, the e2e tests would silently ignore the trust chain validation if the CA.crt field was empty.
I decided it'd be better to *always* perform that check, to catch errors where the Issuer authoer has accidentally forgotten to add set the `IssuerResponse.CA` field.

The ACME issuer doesn't seem to set the CA right now, so I've gone through the ACME e2e tests setting this as an unsupported feature.

Reply:
Bit fan of this approach :smile: